### PR TITLE
Implemented Lecture List

### PR DESCRIPTION
**Pull request by krabagobanjo**
Lecture list functionality is now implemented.

### CODE REVIEW

File: ClassMoodApp/templates/lectureList.html [hunk 1]
```diff
@@ -0,0 +1,39 @@
+{% extends "layout.html" %}
+{% block body %}
+<nav class="navbar navbar-inverse navbar-top">
```
Comment (neilgoel) [inline ClassMoodApp/templates/lectureList.html]: I think the navbar should be in the layout.html file, not the individual pages.

File: ClassMoodApp/static/tsc/controllers/LectureListController.ts [hunk 2]
```diff
@@ -0,0 +1,118 @@
+/// <reference path="../app.ts" />
+/// <reference path="../../../../typings/bootbox/bootbox.d.ts" />
+declare var angular: ng.IAngularStatic;
+module ClassMoodApp {
+    "use strict";
+    export class LectureListController {
+        static $inject = ["$scope", "$http", "$window", "$routeParams"];
```
Comment (neilgoel) [inline ClassMoodApp/static/tsc/controllers/LectureListController.ts]: What is $routeParams used for?

File: ClassMoodApp/Controllers/Lecture.py [hunk 3]
```diff
@@ -32,6 +36,7 @@ def get_live_lecture(class_id):
     if not authUser:
         return render_template('authentication/login.html', error='You are not logged in')
     lectureId = API.get_live_lecture(class_id)
+    print("test")
```
Comment (neilgoel) [inline ClassMoodApp/Controllers/Lecture.py]: Can we remove this?

**Review by neilgoel — APPROVED** [review]
Everything else looks good.

**Comment by krabagobanjo** [comment]
Superseded by #59

**Pull request closed without merging**